app/controllers/problem_reservations_controller.rb

```diff
@@ -0,0 +1,57 @@
+class ProblemReservationsController < ApplicationController
```

Comment:
I considered calling this `ProblemOrderDetailsController` and basing the `:id` on the order detail. That would allow more flexibility for if we ever get to resolving issues with occupancies. However, I ran into an issue because we have a shared module called `ProblemOrderDetailsController` that's shared between the order details and reservations and occupancies for doing the problem order tab.

I could be convinced this should be deeply nested under `/orders/:order_id/order_details/:order_detail_id/problem_reservations/:id` or something like that. Instead I just put it under `/problem_reservations/:id` because it was just simpler.

I do feel like this should be a different controller than the existing ones.